.nf-core.yml

```diff
@@ -1 +1,8 @@
 repository_type: pipeline
+lint:
+  nextflow_config:
+    - config_defaults:
+        - params.logo_file
```

Comment:
That's for fixing the linting error; otherwise nf-core lint will complain about the default paths to the files differing in nextflow.config vs nextflow_schema.json